ghost/admin/app/routes/posts/debug.js

```diff
@@ -8,22 +8,24 @@ export default class Debug extends AuthenticatedRoute {
         let query = {
             id,
             include: [
-                'tags', 
-                'authors', 
-                'authors.roles', 
-                'email', 
-                'tiers', 
-                'newsletter', 
-                'count.conversions', 
-                'count.clicks', 
-                'sentiment', 
-                'count.positive_feedback', 
+                'tags',
+                'authors',
+                'authors.roles',
+                'email',
+                'tiers',
+                'newsletter',
+                'count.conversions',
+                'count.clicks',
+                'sentiment',
+                'count.positive_feedback',
```

Comment:
We can clean this up, and not load all the relations that we had for the analytics page